src/main/java/jewellery/inventory/exception/product/ProductWithoutResourcesException.java

```diff
@@ -0,0 +1,7 @@
+package jewellery.inventory.exception.product;
+
+public class ProductWithoutResourcesException extends RuntimeException {
+    public ProductWithoutResourcesException() {
+        super("The product cannot be created without resources");
```

Comment:
Yes, I did it

Reply:
`NotNull` is not enough in this case because resourcesContent can also be an empty array. This is not good for us as well. That's why I suggested `NotEmpty`. This checks for both null and empty array

Reply:
I did the following: 
@NotEmpty
  private List<UUID> authors;
  @NotNull
  private UUID ownerId;
  @NotEmpty
  List<ResourceQuantityRequestDto> resourcesContent;